docs/src/main/asciidoc/elasticsearch.adoc

```diff
@@ -343,7 +342,24 @@ Quarkus provides support for the Elasticsearch High Level REST Client but keep i
 - It drags a lot of dependencies - especially Lucene -, which doesn't fit well with Quarkus philosophy. The Elasticsearch team is aware of this issue and it might improve sometime in the future.
 - It is tied to a certain version of the Elasticsearch server: you cannot use a High Level REST Client version 7 to access a server version 6.
 
-include::./status-include.adoc[]
+[WARNING]
+====
+Due to the license change made by Elastic for the Elasticsearch REST High Level Client,
+we are keeping in Quarkus the last Open Source version of this particular client, namely 7.10,
+and it won't be upgraded to newer versions.
+
+Given this client was deprecated by Elastic and replaced by a new Open Source Java client,
+the Elasticsearch REST High Level Client extension is considered deprecated and will be removed from the Quarkus codebase at some point in the future.
+
+Note that contrary to the High Level client, we are using the latest version of the Low Level client (which is still Open Source),
+and, while we believe it should work, the situation is less than ideal and might cause some issues.
+Feel free to override the versions of the clients in your applications depending on your requirements,
+but be aware of https://www.elastic.co/blog/elastic-license-v2[the new licence of the High Level REST Client] for versions 7.11+:
+it is not Open Source and has several usage restrictions.
+
+We will eventually provide an extension for the new Open Source Java client but it will require changes in your applications
+as it is an entirely new client.
```

Comment:
Probably better to use the same terminology as the Elasticsearch docs (e.g. https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high.html):

```suggestion
Due to the license change made by Elastic for the Elasticsearch High Level REST Client,
we are keeping in Quarkus the last Open Source version of this particular client, namely 7.10,
and it won't be upgraded to newer versions.

Given this client was deprecated by Elastic and replaced by a new Open Source Java client,
the Elasticsearch High Level REST Client extension is considered deprecated and will be removed from the Quarkus codebase at some point in the future.

Note that contrary to the High Level REST client, we are using the latest version of the Low Level REST client (which is still Open Source),
and, while we believe it should work, the situation is less than ideal and might cause some issues.
Feel free to override the versions of the clients in your applications depending on your requirements,
but be aware of https://www.elastic.co/blog/elastic-license-v2[the new licence of the High Level REST Client] for versions 7.11+:
it is not Open Source and has several usage restrictions.

We will eventually provide an extension for the new Open Source Java client but it will require changes in your applications
as it is an entirely new client.
```